frontend/src/pages/Landing/MyAlerts.js

```diff
@@ -89,51 +95,121 @@ function renderReports(reports) {
   });
 }
 
-function MyAlerts({ reports, newBtn }) {
+export function renderTotal(offset, perPage, activePage, reportsCount) {
+  const from = offset >= reportsCount ? 0 : offset + 1;
+  const offsetTo = perPage * activePage;
+  let to;
+  if (offsetTo > reportsCount) {
+    to = reportsCount;
+  } else {
+    to = offsetTo;
+  }
+  return `${from}-${to} of ${reportsCount}`;
```

Comment:
Let's hide this until pagination is active. With 11 reports the table is displaying all 11 (good, since otherwise one would be inaccessible) but this says `1-10 of 11` which is confusing